OpenRA.Game/Effects/SpriteEffect.cs

```diff
@@ -23,14 +23,14 @@ public class SpriteEffect : IEffect
 		readonly bool visibleThroughFog;
 		readonly bool scaleSizeWithZoom;
 
-		public SpriteEffect(WPos pos, World world, string image, string sequence, string palette, bool visibleThroughFog = false, bool scaleSizeWithZoom = false)
+		public SpriteEffect(WPos pos, World world, string image, string sequence, string palette, int facing = 0, bool visibleThroughFog = false, bool scaleSizeWithZoom = false)
```

Comment:
Is there a special reason that you're adding the `facing` parameter smack in the middle? If you'd put it on the end, people (well, those few with 3rd party dlls) wouldn't need to adjust their code.